test/uk/gov/hmrc/customsservicestatusfrontend/helpers/BaseSpec.scala

```diff
@@ -54,6 +56,7 @@ trait BaseSpec
     with Status
     with HeaderNames
     with ResultExtractors
+    with MockitoSugar
```

Comment:
I don't think we want to add mockito in here when this service is already using scalamock.